src/webhook/webhook.controller.ts

```diff
@@ -0,0 +1,62 @@
+import {
+  Body,
+  Controller,
+  Delete,
+  ForbiddenException,
+  Get,
+  HttpCode,
+  Param,
+  ParseIntPipe,
+  Post,
+  UseGuards,
+} from '@nestjs/common';
+import { WebhookService } from './webhook.service';
+import { UserId } from '../common/decorators/req-user-id.decorator';
+import { JwtAuthGuard } from '../auth/guards/jwt-auth.guard';
+import { CreateWebhookRequest, FetchWebhooksResponse } from './dto';
+import { ApiOperation, ApiTags } from '@nestjs/swagger';
+
+@Controller('webhooks')
+@ApiTags('Webhooks')
+export class WebhookController {
+  constructor(private readonly webhookService: WebhookService) {}
+
+  @UseGuards(JwtAuthGuard)
+  @HttpCode(201)
+  @ApiOperation({ summary: 'Create webhook' })
+  @Post()
+  async createWebhook(
+    @UserId() userId: number,
+    @Body() webhookData: CreateWebhookRequest,
+  ): Promise<void> {
+    try {
+      await this.webhookService.createWebhook(userId, webhookData);
+    } catch {
+      throw new ForbiddenException(
```

Comment:
I would personally use 409 Conflict here, but 403 Forbidden technically seems correct too

Reply:
However, I think you should call `ForbiddenException` within `webhookService` where you are checking for `WEBHOOK_LIMIT`, not in controller

Reply:
Also, consider returning created webhook data - might be useful for the frontend. But it's also fine not to return any data.